src/EditorFeatures/Core/Microsoft.CodeAnalysis.EditorFeatures.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis</RootNamespace>
-    <TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>$(NetVS);netstandard2.0</TargetFrameworks>
```

Comment:
I imagine these could be $(NetRoslyn) instead, since if we're only using them for nullable annotations, and so if NetRoslyn can move ahead of NetVS, then newer would be better there.

Reply:
(same comment applies to other EditorFeatures projects)